contracts/src/integrations/cert/router/EigenDACertVerifierRouter.sol

```diff
@@ -35,10 +36,17 @@ contract EigenDACertVerifierRouter is IEigenDACertVerifierRouter, OwnableUpgrade
 
     /// ADMIN ///
 
-    function initialize(address _initialOwner, address certVerifier) external initializer {
+    function initialize(address _initialOwner, uint32[] memory initRBNs, address[] memory initCertVerifiers)
+        external
+        initializer
+    {
         _transferOwnership(_initialOwner);
-        // Add a default cert verifier at block 0, which will be used for all blocks before the first ABN.
-        _addCertVerifier(0, certVerifier);
+        if (initRBNs.length != initCertVerifiers.length) {
```

Comment:
Should we also check that the initial RBNs are strictly increasing?

Reply:
https://github.com/Layr-Labs/eigenda/pull/1787/commits/f264b3f04cc48d9fba5bc86d37bdc7d4bd253178